modules/ROOT/pages/sap/sap-s4hana-cloud-connector-studio.adoc

```diff
@@ -0,0 +1,129 @@
+
+== Add the Connector to Your Project
+
+Add SAP S/4HANA Odata Cloud Connector to a Mule project to automatically populate the XML code with the connector's namespace and schema location, as well as to add the required dependencies to the project's pom.xml file.
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In *Add Dependencies to Project*, type `SAP S/4HANA OData` in the search field.
+. Click the SAP S/4HANA OData Cloud connector name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+== Configure an Input Source
+
+An input source initiates a flow when a specified condition is met. The act of dragging the connector's operation from the palette onto the Anypoint Studio canvas automatically populates the XML code with the connector namespace and schema location. 
+
+Possible input source operations for this connector are: 
+
+* HTTP Listener +
+Starts your app using a browser or a command such as `curl`.
+* Scheduler +
+Starts your app at timed intervals.
+
+In Studio, drag the input operation you choose from the *Mule Palette* to the *Source* panel in the canvas, and supply the required parameters.
+
+== Global Configuration Elements
+
+To use SAP S/4HANA OData Cloud Connector in a Mule app, you can configure a global SAP S/4HANA configuration element for use by SAP S/4HANA OData Cloud Connector operations in your application.
+
+You can put placeholder values in the `mule-artifact.content` configuration file in the `src/main/resources` folder of your project.
+
+You can either enter your credentials in the global configuration content or 
+reference a configuration file that contains the values for your credentials. 
+Using a configuration file is recommended for simpler maintenance and better reuse of your project. Keeping these values in a separate file is useful if you need to deploy to different environments, such as production, development, and QA, where access credentials might differ.
+
+=== Configure a Global Element
+
+Use the Global Element Properties configuration dialog to configure a global element for SAP S/4HANA OData Cloud Connector:
+
+. In your project, click the *Global Elements* tab at the base of the canvas.
+. In the *Global Configuration Elements* screen, click *Create*:
++
+image::sap/sap-hana-global-config-wizard.png[]
++
+. In the *Choose Global Type* wizard, expand *Connector Configuration*, select *SAP S 4HANA Config*, and click *OK*.
+. Configure the following values in the *General* tab:
+.. In the *Connection* drop-down, select the connection type:
+  * *Basic authentication* +
+  For basic authentication, provide the following required values:
+  ** Service Base URL +
+  Enter the URL of the SAP S/4HANA instance you want to manage.
+  ** Username +
+  Enter the user name used to initialize the session.
```

Comment:
Also GLOBAL, because the intro to the subbullets contains the verb "enter" already, omit "Enter" from all of the descriptions and change them to plain noun fragments:
```suggestion
  The user name used to initialize the session
```
GLOBALLY remove "Enter" and the period; I won't mark all the rest below.